web/themes/custom/badcamp2017/sass/_variables.scss

```diff
@@ -9,4 +9,15 @@ $bg_image: '../images/homepage-bg-image.jpg';
 $bc-blue: #0071bd;
 $bc-green: #6cb229;
 $bc-orange: #ed592a;
-$bc-grey: #585858;
\ No newline at end of file
+$bc-grey: #585858;
+
+// $link-color: #1779ba;
+$link-color: $bc-blue;
+$link-color--hover: lighten($link-color, 20%);
+
+$main-menu-color: black;
```

Comment:
I suppose there are different schools of thought here.  But, black is black, so can we just leave this and move on?